### ISSUE TITLE
HTML title code does not display correctly.

### ISSUE THREAD

**Issue by lhn1703** [WestHealth/pyvis#166]
Issue: title mouse hover text appears as raw html
New pyvis html graph file generated ()
![image](https://user-images.githubusercontent.com/55109682/183485552-9c7039c1-1720-4d89-a495-9f572629e065.png)

Old working pyvis html graph file
![image](https://user-images.githubusercontent.com/55109682/183485739-e57fe753-4faf-4310-a6a9-7fc00ff88a9d.png)

change: 
![image](https://user-images.githubusercontent.com/55109682/183486044-e5b8bdac-d482-4982-93f5-b089b9cdadd1.png)

It appears that changing the stylesheet and script links broke the feature. Request for the write_html() method to automatically use old working version of these links in the generated html file.

**Comment by jhunpingco** [comment]
Related?
https://github.com/WestHealth/pyvis/issues/144

**Comment by lhn1703** [comment]
Yes, it is same issue. Changing the first 2 lines to previous version seemed easier than applying the other workaround, though it would be preferrable to be fixed on pyvis's end.

**Comment by yllowcool** [comment]
Hi fellow community members, I have came up with a solution that is more secured and recommended method by visjs [officially](https://github.com/visjs/vis-network/pull/1378).
Please see the code gist I posted here:
https://gist.github.com/yllowcool/1d21a38a02595b88f99547b98d92411f

Hope it helps 😀

**Comment by Atul-Raj-HUB** [comment]
@yllowcool  Title Mouse hover still appears as raw HTML for me
